generators/app/tasks/fileCreators.js

```diff
@@ -60,7 +67,6 @@ module.exports.createJSConfig = function createJSConfig() {
       '~screens/*': ['./src/app/screens/*'],
       '~config/*': ['./src/config/*'],
       '~constants/*': ['./src/constants/*'],
-      '~redux/*': ['./src/redux/*'],
```

Comment:
If we add a context folder we should add it here

Reply:
I think the I answered this in the previous comment